addons/kommander/1.1.x/kommander.yaml

```diff
@@ -88,5 +88,6 @@ spec:
 
       kommander-ui:
         kubernetesVersionsSelection: '["1.16.4", "1.16.3"]'
+        env: {}
```

Comment:
I wonder why we add them in an empty way here, tbh, do you know?

Reply:
I can remove it again. This is not needed.

Reply:
```suggestion

```